test-case/check-sof-logger.sh

```diff
@@ -62,6 +62,23 @@ etrace_stderr_file=$LOG_ROOT/logger.etrace_stderr.txt
 
 func_lib_check_sudo
 
+sof_alsa_card_found()
+{
+    # note: assumes SOF card names to start with "sof", e.g.
+    #   - /proc/asound/sofsoundwire/id
+    #   - /proc/asound/sofhdadsp/id
+    test -e /proc/asound/sof*/id
+}
+
+wait_for_sof_alsa_card()
+{
+    if poll_wait_for 1 "$MAX_WAIT_FW_LOADING" sof_alsa_card_found ; then
```

Comment:
In a follow-up PR please:
```suggestion
    if poll_wait_for 1 "$1" sof_alsa_card_found ; then
```